pwm.conf

```diff
@@ -0,0 +1,10 @@
+file_client: local
```

Comment:
This salt config is not part of a formula.

Reply:
You are correct. This should be moved to the S3 bucket.